frontend/mulighetsrommet-veileder-flate/tests/veilederflate.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { expect, Page, test } from "@playwright/test";
+import AxeBuilder from "@axe-core/playwright";
+
+test.beforeEach(async ({ page }) => {
+  await page.goto("/");
+});
+
+const sjekkUU = async (page: Page) => {
+  const accessibilityScanResults = await new AxeBuilder({ page })
+    .withTags(["wcag2a", "wcag2aa", "wcag21a", "wcag21aa"])
+    .analyze();
+  expect(accessibilityScanResults.violations).toEqual([]);
+};
+
+const velgFilter = async (page: Page, filternavn: string) => {
+  await page.getByTestId(`filter_radio_${filternavn}`).click();
+  await expect(page.getByTestId(`filter_radio_${filternavn}`)).toBeChecked();
+  await expect(page.getByTestId(`filtertag_${filternavn}`)).toBeVisible();
+};
+
+const tellAntallTiltak = async (count: number, antall: number) => {
```

Comment:
Kunne denne hett `forventAntallTiltak`? Så leses det som `forventAntallTiltak(count, 10)` i testene

Reply:
Hvorfor trengs denne metoden med en for-løkke forresten? Den virker helt unødvendig hvis du bare kan kalle `expect(count).toBe(antall)`.

Reply:
Godt poeng. Prøvde egentlig å lage en funksjon som kunne lagre antallet, så vi slipper å hardkode at det er 5 tiltak, men jeg droppa det. Ser at noe av koden henger igjen 😅

Reply:
Ah, jo! For med den funksjonen så teller for-løkken antall children (men Playwright har ikke .children som Cypress har) for `oversikt_tiltaksgjennomforinger`

Reply:
http://localhost:9323/#?testId=e9fa5836bfcbc49b7c53-7b1efe52151d061e3bfa